facility-manager.html

```diff
@@ -918,10 +980,16 @@
     <li>
 
         <p class="first">
+            Select the data of the user you want to create new objects for
+            ("Polly Stack" is selected in the screenshot).
+            All newly created objects will belong to the selected user.
             Click on <span class="bold">New Project</span>, <span
             class="bold">New Dataset</span> or <span class="bold">New
-            Container</span> icon or <span class="bold">Right-click</span> in
-            the data tree to create a new Project, Dataset or Container.
+            Screen</span> icon or <span class="bold">Right-click</span> in
+            the data tree to create a new Project, Dataset or Screen.
+            Right-clicking on Project and creating Dataset will
+            create a Dataset linked to the Project. Link between the
+            Project and Dataset will belong to the selected user too.
```

Comment:
Might want to mention that you need "Write" privilege etc to be able to create data belonging to other users?